dbscripts/xml/upgrade/3.2.0_preupdate_email_templates.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE data SYSTEM "../../../lib/pkp/dtd/xmlData.dtd">
+
+<!--
+  * dbscripts/xml/upgrade/3.2.0_email_templates.xml
+  *
+  * Copyright (c) 2013-2018 Simon Fraser University
+  * Copyright (c) 2003-2018 John Willinsky
+  * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+  *
+  * Update the email templates tables to remove unused columns, add a new
+	* email_id column and primary key, and update outdated role ids for the
+	* default templates.
+  *
+  -->
+
+<data>
+	<sql>
+		<query>
+			ALTER TABLE email_templates_data
+			ADD email_id BIGINT NOT NULL
+		</query>
+		<query>
+			UPDATE email_templates_data as etd
+			SET etd.email_id = (
+				SELECT et.email_id
+				FROM email_templates as et
+				WHERE etd.email_key = et.email_key
+					AND etd.assoc_type = et.assoc_type
+					AND etd.assoc_id = et.assoc_id
+			)
+		</query>
+		<query>
+			CREATE TABLE email_templates_settings (
+				email_id INT,
+				locale VARCHAR(14) NOT NULL DEFAULT "",
+				setting_name VARCHAR(255) NOT NULL,
+				setting_value text NULL,
+				PRIMARY KEY (email_id,locale,setting_name)
+			)
+		</query>
+		<query>
+			INSERT INTO email_templates_settings (email_id, locale, setting_value, setting_name)
+			SELECT email_id, locale, subject, 'subject'
+			FROM email_templates_data
+		</query>
+		<query>
+			INSERT INTO email_templates_settings (email_id, locale, setting_value, setting_name)
+			SELECT email_id, locale, body, 'body'
+			FROM email_templates_data
+		</query>
+		<query>
+			DROP TABLE email_templates_data
+		</query>
+		<query>
+			ALTER TABLE email_templates
```

Comment:
Likewise, new columns should be left to the schema descriptors (and column changes).